lib/manager/travis/readme.md

```diff
@@ -1 +1,27 @@
-This manager is intended to keep Travis config files (`.travis.yml`) up-to-date. Currently it manages only the `node_js` section of files only.
+This manager is intended to keep Travis config files (`.travis.yml`) up-to-date.
+Currently it manages only the `node_js` section of files only.
+
+An important limitation to note is that Renovate does not currently "understand" Travis's node matrix concept, so it will try to update all found node versions to the latest LTS, e.g.
+
+```diff
+node_js:
+-  - 8.10.0
+-  - 10.10.0
++  - 14.17.4
++  - 14.17.4
+```
+
+Due to this, major updates for Travis are disabled by default.
+If you enable them and use a version matrix, then you will likely need to manually fix any major update PRs raised by Renovate. You can enable them this way in config:
```

Comment:
```suggestion
If you enable major updates and use a version matrix, then you will likely need to manually fix any major update PRs raised by Renovate.
You can major updates them this way in config:
```

Add newline. 😄 

I think it's easier to understand if we say major updates instead of "enable them", but feel free to disagree on this point. 😄